### ISSUE TITLE
Manual quest tracker creation

### ISSUE THREAD

**Issue by myangelkamikaze** [gre4bee/ElectronicObserver#63]
Check if it's possible to make an easy way to edit quest trackers by end users.
Could start with json first to represent the data structure, and possibly make an UI for it later.

**Comment by NaokiSato102** [comment]
I'm sure this feature will come in handy. It's also something that I've been requesting to be implemented in the mainstream.
However, it is a bit difficult to use because of the lack of on-screen instructions on how to use it.
It would be nice if there were easy-to-use in-app explanations.

By the way
Does the mission tracker created by this feature have the ability to feed back to you? If there is, I think it would be a win-win if it could be reflected in the settings, saving the time and effort of updating.

**Comment by myangelkamikaze** [comment]
Agree, it's not really easy to use currently, it's still a work in progress. There are some basic instructions here: https://github.com/gre4bee/ElectronicObserver/wiki/Quest-tracker-manager

I'm not sure what's meant with the feedback question. You mean if we can see the trackers users make? We can't, it's not really a problem since quests don't get added that often, so the trackers are easy enough to make and share.

**Comment by myangelkamikaze** [comment]
I just wasn't sure if that was what you're asking so I wanted to make sure ・。・v
Note that you can write in Japanese if you want.

**Comment by NaokiSato102** [comment]
では遠慮なく。
作成した任務データを、簡単にシェアできれば、神風氏が更新するまで間を持たせることができるほか、そのデータを（精査の上）本体に組み込むこともできるのではと思いました。

**Comment by myangelkamikaze** [comment]
Ah, okay, so I understood it correctly.

Automatic tracker submission would probably take longer than making trackers does, and I get help with tracker making on discord too, so that wasn't considered at all.

**Comment by NaokiSato102** [comment]
そういえばDiscordありましたね

**Comment by myangelkamikaze** [comment]
・。・ｖ
![image](https://user-images.githubusercontent.com/40002167/149616859-d4b1a162-a05b-461f-8134-1dffb28228a4.png)
